### PR TITLE
Adds default includes for boilerplate content 

### PR DESCRIPTION
**Pull request by msagolj**
Adds default includes for boilerplate content

### CODE REVIEW

File: paths.json [hunk 1]
```diff
@@ -2,5 +2,9 @@
   "mappings": [
     "/content/aem-boilerplate/:/",
     "/content/aem-boilerplate/configuration:/.helix/config.json"
+  ],
+  "includes": [
+    "/content/aem-boilerplate/",
+    "/content/dam/aem-boilerplate/"
```
Comment (mhaack) [inline paths.json:8]: Should we really include /content/dam by default? I would rater put that in docs only. As this is only needed if assets are published directly outsite of the page context.

Comment (buuhuu) [reply paths.json:8]: Yes exactly. Please remove that.

**Comment by buuhuu** [comment]
We must also remove the /conf/<sitename> setting from the assets folder.

File: paths.json (hunk 1 above)
Comment (msagolj) [reply paths.json]: removed

**Comment by msagolj** [comment]
> We must also remove the /conf/ setting from the assets folder.

are you referring to this in franklin-examples  ?
https://git.corp.adobe.com/CQ/sites-franklin/blob/main/content/sites-franklin-examples/jcr_root/content/dam/franklin-examples/.content.xml#L6

**Comment by msagolj** [comment]
> We must also remove the /conf/ setting from the assets folder.

@buuhuu are you referring to this in sites-franklin-examples ? 
https://git.corp.adobe.com/CQ/sites-franklin/blob/main/content/sites-franklin-examples/jcr_root/content/dam/franklin-examples/.content.xml#L6

**Comment by buuhuu** [comment]
> are you referring to this in sites-franklin-examples ?
yes, @msagolj

**Comment by msagolj** [comment]
> > are you referring to this in sites-franklin-examples ?
> > yes, @msagolj

opened PR in sites-franklin:
https://git.corp.adobe.com/CQ/sites-franklin/pull/148

**Pull request merged**